openedx/core/lib/js_utils.py

```diff
@@ -68,3 +52,32 @@ def escape_json_dumps(obj, cls=EdxJSONEncoder):
     encoded_json = json.dumps(obj, ensure_ascii=True, cls=cls)
     encoded_json = _escape_json_for_html(encoded_json)
     return encoded_json
+
+
+def escape_js_str(js_str):
```

Comment:
Nit: can you rename this to ``escape_javascript`` or ``escape_javascript_string``. We try not to use abbreviations when naming variables.